llms/llama/llama.py

```diff
@@ -384,6 +385,7 @@ def load_model(model_path):
     parser.add_argument(
         "--temp", type=float, default=0.0, help="The sampling temperature"
     )
+    parser.add_argument("--eos", type=str, default="\n\n\n", help="End of sequence")
```

Comment:
Any reason to use an argument here instead of getting the EOS token from the tokenizer?

Reply:
Sometimes we want to stop generating on a predefined sequence, instead of EOS from the tokenizer.

Reply:
End of sequence may be application concerns, but it should not be the main focus of the example. I think we could include it as part of mlx-lm package support. However, personally, I feel we should keep the original llama example concise demonstrates how to run llama via mlx without adding too much unnecessary noise there.

Reply:
I agree with @mzbac on this. This example is definitely not meant to be feature complete, it's meant to be simple and readable. New features such as this one make a lot more sense in mlx-lm.

Reply:
Makes sense. I'll remove change that adds new arg and supporting code. requirements change should stay since it breaks conversion script. Let me know if you have any other concerns.

Reply:
Perfect thanks!